LKMEX-whitelist/combased-lkmex.md

```diff
@@ -0,0 +1,30 @@
+Name of project:
+
+Combased (Combeys, Combots)
+
+https://explorer.elrond.com/nfts/COMBEYS-bc640d
+
+Description of the project: 
+
+We are the first e-commerce project on Elrond and official Shopify partner. We provide quality blockchain services as well. Also we have our NFT collections: Combeys, Combots
+
+Purpose: 
+
+We have a transparent CDO wallet, which has LP for farming LKMEX. That LP is increased from our e-commerce and web3 development services revenue. 
+
+Address of Smart Contract:
+
+erd1qqqqqqqqqqqqqpgqss2kwutkwdvpyey60u7felml0jmrqp6hmkdqadcl5z
```

Comment:
Is this a multi sig contract ? We need a well defined code in order to whitelist.

Reply:
It is not multi sig. Should it be? Ok I will attach the code (Krogan fork). Can we edit the code later?

Reply:
@sasurobert hello? :)

Reply:
We need access to the code and we need to see if this can be used for wrong purposes.